cli/src/semgrep/commands/install.py

```diff
@@ -29,6 +29,16 @@
 
 
 def determine_semgrep_pro_path() -> Path:
+    # The SEMGREP_PRO_PATH env var is set for the non-root Docker image to install
+    # the Pro Engine binary in the semgrep user's $HOME/bin directory where it has
+    # the proper permissions to do so.
+    SEMGREP_PRO_PATH = os.getenv("SEMGREP_PRO_PATH")
+
+    if SEMGREP_PRO_PATH:
+        path = Path(SEMGREP_PRO_PATH) / "semgrep-core-proprietary"
+        path.parent.mkdir(parents=True, exist_ok=True)
+        return path
```

Comment:
I would love to see test output for what happens if the user's `SEMGREP_PRO_PATH` is invalid (hoping for a clear error message)

Reply:
Hey @emjin, if the path is invalid the docker image crashes much like it already does when trying to install the pro binary as a non-root user. I did just add some validation however to exit more gracefully if the path is invalid.